### DISCUSSION TITLE
Migration to Codeberg and Organization Membership Changes

### DISCUSSION THREAD

**Discussion by generic-pers0n** [tenacityteam/tenacity-legacy#739] [Announcements]
Hi everyone! I'm the new maintainer of Tenacity, and we have some news to share!

Almost a month ago, I was made an owner of the Tenacity team organization here on GitHub. Since then, I moved the Saucedacity repository from my own Saucedacity organization to under this organization. This was temporary, and the final repository will be renamed to 'tenacityteam/tenacity' after rebranding. Afterwards, we will move the repository to Codeberg, and development will officially continue there.

The migration process has already begun, and we've already migrated a few repositories, including this repository too. For those repositories, future development will continue on Codeberg rather than GitHub. For any repositories not on Codeberg, we will most likely not continue development on them, or they are waiting to be moved (most notably with https://github.com/tenacityteam/saucedacity).

However, migrating these repositories to Codeberg means migrating them under a new organization. As of this time of writing, the organization contains only me and @caughtquick as owners and no other (regular) members. Therefore, I would like to announce a new project membership policy for former Tenacity developers, contributors, and maintainers. Before I get started, however, I would like to assure you that **there will be no membership changes with the Tenacity organization on GitHub unless explicitly requested by a member for their own membership status and not for someone else's**. This announcement will only apply to the Tenacity organization **on Codeberg** (https://codeberg.org/tenacityteam).

The policy goes as follows:

* If you were a Tenacity contributor, maintainer, or owner of the GitHub organization **and** would like to get re-involved in future Tenacity development, please contact me, @generic-pers0n, requesting the appropriate permissions. If you do specify the permissions you would like, I will either give you the same permissions as you had in the GitHub organization or basic membership permissions.

And yep, that's the entire policy! I've had intentions of doing this for the new Codeberg organization for some time. And remember, unless explicitly requested by a member for their own permissions, **I will not change any permissions in the GitHub organization; all is only applicable to the Codeberg organization**.

**UPDATE**: I will make an exception for changing permissions to the GitHub organization, if people are added. Existing members, however, will continue to retain their current permissions unless they explicitly request that I change their permissions for themselves and only themselves.

I'm really looking forward to Tenacity development restarting, which is at any moment now. Once https://github.com/tenacityteam/saucedacity/pull/81 is merged, I will rename and migrate the repository as appropriate, and we'll be on our way to releasing Tenacity 1.3! Finally, if you were a former Tenacity contributor, developer, or maintainer and would like to get re-involved, I hope to see you on Codeberg! :D

-- Avery King

**Edit**: If you have questions, comments, or concerns, feel free to post them below. Feedback is always appreciated!

**Comment by leio** [comment]
Glad to see continued efforts in this kind of a audio editor and good luck in your endeavor on Codeberg.
Given that I have not been active in tenacity for a while, I won't be joining another forge just for that at this point of time.

If you have any wxWidgets or GTK issues I can help with in chat, feel free to reach out to me on IRC or with appropriate matrix bridges. I will still be hanging out in #tenacity and co channels on Libera for the time being as well. Though those channels don't seem to see much action - maybe the Matrix bridge goes elsewhere or isn't set up or something.

I wish you a lot of tenacity in developing Tenacity!

**Reply by n0toose** [reply]
Hey, just wanting to get this out there, somewhere: Your contributions were valuable and you rock.

**Comment by generic-pers0n** [comment]
I actually need to make a quick update to this announcement:

While I won't be changing permissions to anyone unless requested for themselves, **I will be adding people to both the Codeberg and GitHub organizations. I will not, however, remove people from the GitHub organization that joined prior to this point**.
